src/components/Game/GameScreen.tsx

```diff
@@ -91,13 +90,43 @@ function GameScreen({
         setTravelLegs([event.startLocation])
         setEndLocation(event.endLocation)
         setStartTime(eventStartDate)
+        fetchAvailableModes(event.startLocation)
     }, [event])
 
     useEffect(() => {
         setTimeDescription(formatTimeForEndOfGame(currentTime, startTime))
         window.scrollTo(0, document.body.scrollHeight)
     }, [currentTime])
 
+    const fetchAvailableModes = async (location: StopPlace) => {
+        const modes: QueryMode[] = [
+            QueryMode.BUS,
+            QueryMode.METRO,
+            QueryMode.TRAM,
+            QueryMode.RAIL,
+            QueryMode.WATER
+        ]
+        console.log(modes)
+
+        const departurePromises = modes.map(mode => getDepartures(location.id, mode, currentTime))
+        const walkableStopsPromise = getWalkableStopPlaces(location)
+        const results = await Promise.all([...departurePromises, walkableStopsPromise])
+        const walkableStops = results.pop() as StopPlace[]
+
+        const validModes = results
+            .map((deps, index) => (deps as Departure[]).length > 0 ? modes[index] : null)
+            .filter(isTruthy)
+
+        if (walkableStops.length > 0) {
+            validModes.unshift(QueryMode.FOOT)
+        }
+
+        setAvailableModes(validModes)
+        console.log(validModes)
```

Comment:
Kan fjerne console.log

Reply:
Takk for observasjonen! Gjorde det i neste branch nå